### DISCUSSION TITLE
left-1/2, left-1, left-2... not working

### DISCUSSION THREAD

**Discussion by nerdess** [tailwindlabs/tailwindcss#3489] [Help]
I am trying to add left:50% to an absolute positioned element. But none of the left properties except left-0 are doing anything. So left-1/2, left-1, left-2.... are all not working.

this:
![Bildschirmfoto 2021-02-01 um 09 35 23](https://user-images.githubusercontent.com/711925/106433704-d48c2080-6470-11eb-8d08-b920d408c787.png)

results in this:
![Bildschirmfoto 2021-02-01 um 09 32 28](https://user-images.githubusercontent.com/711925/106433576-adcdea00-6470-11eb-8183-30b4048ebdc2.png)

i checked my tailwind config, all seems fine, nothing gets overwritten:

```
module.exports = {
  theme: {
    backgroundColor: theme => ({
      'foo': '#F7A600',
    }),
    colors: {
      'foo': '#F7A600',
    },
    extend: {
      colors: {
        'foo': '#F7A600',
      },
      fontSize: {
        '7xl': '5rem',
        'base': '18px',
      },
    },
  },
  variants: {},
  plugins: [],
}
```

any hints would be great!

**Comment by simonswiss** [comment]
Hey!

This seems to be working properly: https://play.tailwindcss.com/ftWqHyfppW

Are you able to reproduce this issue in a Tailwind Play or maybe share a GitHub repo?

**Reply by nerdess** [reply]
thx but i can't share the repo since it is a commercial project :(

tried to replicate as much as i could in tailwind play and left-1/2 works perfectly fine there.

locally however left-1/2 is still not working. same with a few other directives such as m-0.5 (m-1 is working though). 

weird stuff.
